modxcms/revolution
# Enable SVG thumbnail preview for image TVs

### What does it do?
Skips thumbnailing of SVGs via phpThumb.

### Why is it needed?
Fixes inability to preview image TVs containing SVG images.

### How to test
1. Run `grunt build` at `_build/templates/default` and ensure your browser and modx caches are cleared.
2. Save two SVG images, associating one with the default Filesystem source and the other with an alternate media source.
3. Create two image TVs, one for each source you saved your SVGs to.
4. Assign the TVs to a Template and insert your images into a Resource assigned that Template to verify the preview functionality.

### Related issue(s)/PR(s)
Resolves #16270 and is a port of #15865.
